app/forms/registration_form.rb

```diff
@@ -50,38 +46,5 @@ def administrative_tags
     params[:tag].filter { |t| !t.start_with?('Process : Content Type') }
   end
 
-  private
-
   attr_reader :params
-
-  def dro_type
-    case content_type_tag
-    when 'Image'
-      Cocina::Models::ObjectType.image
-    when '3D'
-      Cocina::Models::ObjectType.three_dimensional
-    when 'Map'
-      Cocina::Models::ObjectType.map
-    when 'Media'
-      Cocina::Models::ObjectType.media
-    when 'Document'
-      Cocina::Models::ObjectType.document
-    when /^Manuscript/
-      Cocina::Models::ObjectType.manuscript
-    when 'Book (ltr)', 'Book (rtl)'
-      Cocina::Models::ObjectType.book
-    when 'Webarchive-seed'
-      Cocina::Models::ObjectType.webarchive_seed
-    when 'Geo'
-      Cocina::Models::ObjectType.geo
-    else
-      Cocina::Models::ObjectType.object
-    end
-  end
-
-  # helper method to get just the content type tag
-  def content_type_tag
-    content_tag = params[:tag].find { |tag| tag.start_with?('Process : Content Type') }
-    content_tag.split(':').last.strip
-  end
 end
```

Comment:
we can rid of all these special code to parse content type and reading direction out of a content tag, and instead just pass them in as form parameters